mkdocs.yml

```diff
@@ -205,7 +205,7 @@ hooks:
 - 'docs/plugins/main.py'
 
 plugins:
-- social
+# - social
```

Comment:
Can't we remove it?

Reply:
Oops, meant to leave this, I just comment this out for local builds